### PR TITLE
Add publish and versioning instructions to readme

### PR DESCRIPTION
**Pull request by Tehsurfer**
More detailed release documentation coming soon

### CODE REVIEW

**Review by nickerso — APPROVED** [review]
this looks to provide the required information - but would be good to have someone who actually runs this stuff to confirm the instructions are correct :)

**Comment by Tehsurfer** [comment]
Thanks @nickerso! Yeah we will check in with Alan W to see if there is anything I'm missing

File: README.md [hunk 1]
```diff
@@ -86,3 +86,71 @@ npm run serve
...
+Dist tags can be used like so:
+```
+npm publish --tag <tag>
+```
+
+`<tag>` is the dist tag to add. In example: `npm publish --tag beta`
+
+This builds the package to be digested by other vue packages by using
+```
+import { ScaffoldVuer } from "@abi-software/scaffoldvuer/src/components/index.js";
```
Comment (alan-wu) [inline README.md:115]: We are building the mapintegratedvuer with the source directly but it should normally be done using 
`import { ScaffoldVuer } from "@abi-software/scaffoldvuer;` in other cases. 

css should be imported  as well when including scaffoldvuer using `import { ScaffoldVuer } from "@abi-software/scaffoldvuer;`.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Looks good but some small change is required. Please see the comment

**Review by alan-wu — APPROVED** [review]

**Pull request merged**